<?xml version="1.0"?>
<launch>


<include file="$(find joy)/launch/joytestps4.launch"/>
<!-- <node pkg="gamepad" type="gamepad_node" name="gamepad_node" output="screen">	
</node>-->

<node pkg="conversion" type="quatCo.py" name="quatCo" output="screen">	
</node>

<node pkg="conversion" type="quatCo2.py" name="quatCo2" output="screen">	
</node>
<!-- keiganモーター制御/オドメトリ算出ノード -->
<node pkg="pypro" name="pitakuru_wheels" type="pitakuruWheels.py">
</node>

<include file="$(find rplidar_ros)/launch/rplidar_pit.launch" />
<node pkg="tf" type="static_transform_publisher" name="laser_frame" args="0 0 0 0 0 0 1 base_link laser 100" />

<include file="$(find hector_slam_launch)/launch/tutorial.launch"/>

<node pkg="lidarPeople" type="main.py" name="lidarPeople_node" output="screen">	
</node>
<!-- <include file="$(find rplidar_ros)/launch/rplidar.launch" />
<node pkg="tf" type="static_transform_publisher" name="laser_frame" args="0 0 0 0 0 0 1 base_link laser 10" />
<include file="$(find gmapping)/launch/slam_gmapping_pr2.launch" />-->

</launch>
